src/main/java/com/gw2auth/oauth2/server/service/application/client/ApplicationClientService.java

```diff
@@ -1,9 +1,13 @@
 package com.gw2auth.oauth2.server.service.application.client;
 
+import com.gw2auth.oauth2.server.service.OAuth2ClientApiVersion;
+
 import java.util.*;
 
 public interface ApplicationClientService {
 
+    OAuth2ClientApiVersion CURRENT_CLIENT_API_VERSION = OAuth2ClientApiVersion.V0;
```

Comment:
Shouldn't this be `V1`?

Reply:
`CURRENT` is used for new client creations, but since this commit changes a lot in all parts of the code I'd like to get the initial rollout without switching the version for new clients yet. 

I'll also update the developer wiki prior to rolling out V1 as default.